libs/components/lookup/testing/src/modules/lookup/lookup-harness.spec.ts

```diff
@@ -146,23 +201,48 @@ function testSingleSelect(dataSkyId: string) {
     });
 
     // First, set a value on the autocomplete.
-    await lookupHarness?.enterText('d');
-    await lookupHarness?.selectSearchResult({
+    await (await lookupHarness.getControl()).setValue('d');
+    await lookupHarness.selectSearchResult({
+      text: 'Leonard',
+    });
+    await expectAsync(
+      (await lookupHarness.getControl()).getValue(),
+    ).toBeResolvedTo('Leonard');
+
+    // Now, clear the value.
+    await (await lookupHarness.getControl()).clear();
+    await expectAsync(
+      (await lookupHarness.getControl()).getValue(),
+    ).toBeResolvedTo('');
+  });
+
+  it('should clear the input value - deprecated', async () => {
+    const { lookupHarness } = await setupTest({
+      dataSkyId: dataSkyId,
+    });
+
+    // First, set a value on the autocomplete.
+    await (await lookupHarness.getControl()).setValue('d');
+    await lookupHarness.selectSearchResult({
       text: 'Leonard',
     });
-    await expectAsync(lookupHarness?.getValue()).toBeResolvedTo('Leonard');
+    await expectAsync(
+      (await lookupHarness.getControl()).getValue(),
+    ).toBeResolvedTo('Leonard');
 
     // Now, clear the value.
-    await lookupHarness?.clear();
-    await expectAsync(lookupHarness?.getValue()).toBeResolvedTo('');
+    await lookupHarness.clear();
+    await expectAsync(
+      (await lookupHarness.getControl()).getValue(),
```

Comment:
For all the specs in this PR, could you set this to a variable and reuse it throughout the spec to reduce clutter?
```
const control = await lookupHarness.getControl();
```

Better yet, you could try returning this from the `setupTest` function.